packages/admin-panel/src/authentication/reducer.js

```diff
@@ -42,8 +42,8 @@ const logoutStateUpdater = (payload, currentState) => ({
   emailAddress: currentState.emailAddress, // But, remember user's email address
 });
 
-const isBESAdmin = policy => {
-  return new AccessPolicy(policy).allowsSome(undefined, BES_ADMIN_PERMISSION_GROUP);
+export const reduceIsVizBuilderUser = policy => {
```

Comment:
Part of me feels this function should just be a generic util, rather than within this reducer file? Especially if we're gonna remove the whole `isVizBuilderUser` from the redux state entirely 👍